### ISSUE TITLE
Bug in PySide6 prevents application from launching

### ISSUE THREAD

**Issue by mmoghaddam385** [SolderedElectronics/Soldered-Image-Converter#3]
Operating System: Windows 11
Python Version: 3.9.13

I tried running the executable in the v1.0.0 release and the application never opened. I saw that the application was "running" in task manager but the window never opened.

Next I tried running it as a python script: I cloned the repo, installed the dependencies and ran `python main.py`.
Again the application was "running" but no window appeared.
I tried this on both the `main` and `dev` branches.

I came across [this StackOverflow question](https://stackoverflow.com/questions/77736041/pyside6-quiloader-doesnt-show-window) which suggested initializing `QUiLoader()` before calling ` QApplication(sys.argv)`. This suggestion worked and the application launched. The answer to that question mentioned it might be a bug in the PySide6 library, so I reverted my change and updated PySide6 to version 6.8.1 and the application was able to launch correctly.

 Basically I'm suggesting to update PySide6 to version 6.8.1 (currently the latest version) to fix this bug.

**Comment by rsoric** [comment]
Hi @mmoghaddam385 

thank you for the suggestion, this is very helpful to us as our primary focus is on embedded and we're still learning about development for desktop applications :)  We will look into this and it will likely be included in the next official release. This is a typical case of 'it works on my machine' but of course it's our goal to make this app available to anyone.

Will write back here when there's updates, likely after the holidays
-Rob

**Comment by fubby14** [comment]
i am having the same issue did this ever get fixed

**Comment by annoholiker** [comment]
Same for me on a windows 10 machine

**Comment by rsoric** [comment]
Hi everyone,

really sorry about this issue, as writing desktop software for us is a new feat, this is one of the consequences unfortunately. 

I will have someone check out this issue in more detail and see if we can publish a quick fix, but as a long-term solution it's likely we will create a web-based version of this app which will not require an installation at all. 

Here is a web tool you can use to convert in the meantime: https://solderedelectronics.github.io/Inkplate-image-converter/ 

Best
-Rob

**Comment by rsoric** [comment]
Hi @mmoghaddam385  @annoholiker  @fubby14 ,

this app has been ported to the web! Check it out: https://soldered.com/image-converter/

Gonna close this issue as we are going to archive this repo

Best
-Rob
